client/dashboard/profile-wizard/style.scss

```diff
@@ -311,3 +315,31 @@
 		margin: 0;
 	}
 }
+
+.components-modal__frame.woocommerce-profile-wizard__usage-modal {
+	min-width: 600px;
+
+	.components-modal__header {
+		border-bottom: 0;
+		margin-bottom: 0;
+	}
+
+	.woocommerce-profile-wizard__usage-wrapper {
+		flex-grow: 1;
+		display: flex;
+		flex-direction: column;
+
+		a {
+			color: $studio-gray-60;
+		}
+
+		button.is-primary {
```

Comment:
Could we remove the bottom margin on this button?  Looks like a bit more spacing on the bottom of this modal because of the button margin.